src/main/java/io/cdap/plugin/gcp/dataplex/common/config/DataplexBaseConfig.java

```diff
@@ -75,11 +75,7 @@ public class DataplexBaseConfig extends PluginConfig {
   @Description("The existing connection to use.")
   protected GCPConnectorConfig connection;
 
```

Comment:
Let's add a java doc saying it returns the reference name or the normalized FQN, since it is a public method